### PR TITLE
Build Page for Baklava Launch Part 1 

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

Implements the changes in design and content needed on build page for Baklava launch. 

HighLights

1.  `Transceive` an animation showing communication between a gold coin and a green coin representing a validator and an application. 
2.  Scroll based highlighting of the stack from top to bottom in sync with a little animated vector of it
3. More amazing things like the leaderboard

This PR contains work in progress for the Leaderboard and as such omits the leaderboard from showing on the page at all. follow up will make it show for dev.celo.org per conversation with @cmcewen as for the intent is not to show leaderboard on main site until there is some real action in there but on dev we want it for the mini stake off. 

### Tested
Screen Shot 2019-11-25 at 8 52 33 PM
Screen Shot 2019-11-25 at 8 52 43 PM
Screen Shot 2019-11-25 at 8 52 54 PM
Screen Shot 2019-11-25 at 8 53 07 PM
Screen Shot 2019-11-25 at 8 53 21 PM
Screen Shot 2019-11-25 at 8 53 35 PM
Screen Shot 2019-11-25 at 8 53 45 PM

### Other changes

better handling of `#links` and scrolling to them nicely
A new Layout system called `BookLayout` which uses the left column for titles and the center for main content. 

### Related issues
see also https://github.com/celo-org/celo-monorepo/pull/1864 which is based on this branch

### Backwards compatibility

contains some big differences in content and layout

### CODE REVIEW

File: packages/web/pages/_app.tsx [hunk 1]
```diff
@@ -3,33 +3,31 @@ import getConfig from 'next/config'
...
 import Footer from 'src/shared/Footer.3'
-import { scrollTo } from 'src/utils/utils'
+import { HEADER_HEIGHT } from 'src/shared/Styles'
 import Sentry, { initSentry } from '../fullstack/sentry'
 import { appWithTranslation } from '../src/i18n'
 
 config({ ssrReveal: true })
 class MyApp extends App {
   componentDidMount() {
-    if (canTrack()) {
```
Comment (cmcewen) [inline packages/web/pages/_app.tsx:17]: Any reason to remove this?

File: packages/web/pages/_app.tsx [hunk 2]
```diff
@@ -72,3 +70,9 @@ function checkH1Count() {
     }
   }, 500)
 }
+
+function hashScroller(id: string) {
```
Comment (cmcewen) [inline packages/web/pages/_app.tsx:76]: very very very small nit but i think it's a little easier to parse when functions are above the component

File: packages/web/src/dev/BuildPage.tsx [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
     return { namespacesRequired: ['common', 'dev'] }
   }
 
   render() {
-    const { t } = this.props
     return (
-      <View>
+      <View
+        style={{
+          // @ts-ignore
```
Comment (cmcewen) [inline packages/web/src/dev/BuildPage.tsx]: think this could go in react-native-aug.d.ts

File: packages/web/src/dev/BuildPage.tsx [hunk 4]
```diff
@@ -29,49 +30,10 @@ class BuildPage extends React.PureComponent<I18nProps> {
...
-            buttonOne={{ title: t('readMore'), href: CeloLinks.docsOverview }}
-            buttonTwo={{ title: t('seeCode'), href: CeloLinks.blockChainRepo }}
-          >
-            <Li style={textStyles.readingOnDark}>{t('proof.permissionless')}</Li>
-            <Li style={textStyles.readingOnDark}>{t('proof.rewardsWeighted')}</Li>
-            <Li style={textStyles.readingOnDark}>{t('proof.onChain')}</Li>
-          </StackSection>
-        </View>
-        <DeveloperUpdates />
+        <FullStack />
```
Comment (cmcewen) [inline packages/web/src/dev/BuildPage.tsx:28]: 💯

File: packages/web/src/dev/Racer.tsx [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+      </View>
+    </View>
+  )
+})
+
+const styles = StyleSheet.create({
+  line: {
+    height: 2,
+    width: 1,
+
```
Comment (cmcewen) [inline packages/web/src/dev/Racer.tsx]: extra space

File: packages/web/src/dev/Racer.tsx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  },
+  lineGroup: {
+    marginTop: 5,
+    alignItems: 'center',
+  },
+  animatable: {
+    animationDuration: '2s',
+    animationDelay: '100ms',
+    animationIterationCount: 1,
+    // animationTimingFunction: 'ease-in',
```
Comment (cmcewen) [inline packages/web/src/dev/Racer.tsx]: comment

**Review by cmcewen — COMMENTED** [review]
V meaty - page looks really cool

One thing I found was this jump:
jump

File: packages/web/pages/_app.tsx (hunk 1 above)
Comment (aaronmgdr) [reply packages/web/pages/_app.tsx:17]: im actually scratching my head as to why this was changed.

File: packages/web/pages/_app.tsx (hunk 2 above)
Comment (aaronmgdr) [reply packages/web/pages/_app.tsx:76]: ah good to know, ive found im pretty indifferent with a 1% lean towards bottom but i do find having all below or all above seems nicest.

File: packages/web/pages/_app.tsx (hunk 1 above)
Comment (aaronmgdr) [reply packages/web/pages/_app.tsx:17]: i probably messed up a merge

**Comment by aaronmgdr** [comment]
That jump is unfortunate. I wasn't able to find a way for it to stick at the exact right spot every time.

**Review by cmcewen — APPROVED** [review]

**Pull request merged**